plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/PomXmlScanner.java

```diff
@@ -74,10 +82,15 @@ public void endElement (String uri, String localName, String qName)
     } else if (foundBuild && "groupId".equals(localName)) {
       foundGroupId = false;
       if ("com.google.appengine".equals(groupIdContents.toString())) {
-        DocumentLocation start = new DocumentLocation(lineNumber,
-            columnNumber - qName.length() - 2);
+        foundOldPlugin = true;
+      }
+    } else if (foundOldPlugin && "artifactId".equalsIgnoreCase(localName)) {
```

Comment:
I think there is an issue here because in Maven, `artifactId` can appear before `groupId`.